packages/manager/apps/telecom/src/app/telecom/pack/migration/ont-shipping/pack-migration-ont-shipping.controller.js

```diff
@@ -0,0 +1,91 @@
+import get from 'lodash/get';
+import head from 'lodash/head';
+import map from 'lodash/map';
+
+export default class TelecomPackMigrationOntShippingCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    TucPackMigrationProcess,
+    OvhContact,
+    OvhApiPackXdsl,
+    TucToast,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.TucPackMigrationProcess = TucPackMigrationProcess;
+    this.OvhContact = OvhContact;
+    this.OvhApiPackXdsl = OvhApiPackXdsl;
+    this.TucToast = TucToast;
+  }
+
+  $onInit() {
+    this.process = null;
+    this.ovhContactOptions = {
+      options: {
+        allowCreation: true,
+        allowEdition: false,
+      },
+    };
+    this.loading = {
+      init: true,
+    };
+
+    this.process = this.TucPackMigrationProcess.getMigrationProcess();
+
+    this.process.ontShipping = {
+      address: null,
+    };
+
+    // Initialize ONT address shipping
+    return this.OvhApiPackXdsl.v6()
+      .shippingAddresses({
+        packName: this.process.pack.packName,
+        context: 'voipLine',
+      })
+      .$promise.then((ontShippingAddresses) => {
+        this.ovhContactOptions.customList = map(
+          ontShippingAddresses,
+          (ontShippingAddress) =>
+            new this.OvhContact({
+              address: {
+                line1: ontShippingAddress.address,
+                city: ontShippingAddress.cityName,
+                country: ontShippingAddress.countryCode,
+                zip: ontShippingAddress.zipCode,
+              },
+              firstName: ontShippingAddress.firstName,
+              lastName: ontShippingAddress.lastName,
+              id: ontShippingAddress.shippingId,
+            }),
+        );
+
+        this.process.ontShipping.address = head(
+          this.ovhContactOptions.customList,
+        );
+      })
+      .catch((error) => {
+        const msgErr = `${this.$translate(
+          'telecom_pack_migration_ont_shipping_addresses_error',
+        )} ${get(error, 'data.message', '')}`;
```

Comment:
Same no need get lodash
```suggestion
        )} ${error?.data?.message}`;
```